src/components/transactions/send/Send.tsx

```
@@ -135,58 +139,75 @@ export function Send({ setSection }: Props) {
+                                        type="button"
+                                        onClick={(e) => {
+                                            e.preventDefault();
+                                            e.stopPropagation();
+                                            clearErrors('amount');
+                                            setValue('amount', numericBalance, { shouldValidate: true });
+                                        }}
+                                    >
+                                        {t('send.max')}
+                                    </Button>
+                                )}
+                            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)